pkg/apis/capabilities/v1alpha1/api_types.go

```diff
@@ -39,15 +39,18 @@ type APISelectors struct {
 }
 
 // APIStatus defines the observed state of API
+// +k8s:openapi-gen=true
 type APIStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
```

Comment:
Should we keep this comment auto generated?? 

https://book.kubebuilder.io/beyond_basics/generating_crd.html

returns 404

Reply:
We can remove it because it does not work anymore. It was useful because in case we wanted to add custom validation there explained how